mirobo/containers.py

```diff
@@ -39,6 +39,115 @@ def pretty_time(x: int) -> datetime:
 }
 
 
+class AirPurifierStatus:
+    """Container for status reports from the air purifier."""
+    def __init__(self, data: Dict[str, Any]) -> None:
+        # ['power', 'aqi', 'humidity', 'temp_dec',
+        #  'mode', 'led', 'led_b', 'buzzer', 'child_lock',
+        #  'limit_hum', 'trans_level', 'bright',
+        #  'favorite_level', 'filter1_life', 'act_det',
+        #  'f1_hour_used', 'use_time', 'motor1_speed']
+        self.data = data
+
+    @property
+    def power(self) -> str:
+        return self.data["power"]
+
+    @property
+    def is_on(self) -> bool:
+        return self.power == "on"
+
+    @property
+    def aqi(self) -> int:
+        return self.data["aqi"]
+
+    @property
+    def humidity(self) -> int:
+        return self.data["humidity"]
+
+    @property
+    def temperature(self) -> float:
+        return self.data["temp_dec"] / 10.0
+
+    @property
+    def mode(self) -> str:
+        # auto, silent, favorite, medium, high, strong, idle
+        return self.data["mode"]
+
+    @property
+    def led(self) -> bool:
+        return self.data["led"] == "on"
+
+    @property
+    def led_b(self) -> str:
+        brightness_levels = {
+            0: "bright",
+            1: "dim",
+            2: "off"
+        }
+        return brightness_levels[self.data["led_b"]]
+
+    @property
+    def buzzer(self) -> bool:
+        return self.data["buzzer"] == "on"
+
+    @property
+    def child_lock(self) -> str:
+        return self.data["child_lock"]
+
+    @property
+    def limit_hum(self) -> str:
+        return self.data["limit_hum"]
+
+    @property
+    def trans_level(self) -> str:
+        return self.data["trans_level"]
+
+    @property
+    def bright(self) -> int:
+        return self.data["bright"]
+
+    @property
+    def favorite_level(self) -> str:
+        # Favorite level used when the mode is `favorite`. Between 0 and 16.
+        return self.data["favorite_level"]
+
+    @property
+    def filter_life_remaining(self) -> int:
+        return self.data["filter1_life"]
+
+    @property
+    def act_det(self) -> bool:
+        return self.data["act_det"] == "on"
```

Comment:
Any description for this , limit_hum and trans_level? The names are not really descriptive.